### DISCUSSION TITLE
Duvida referente a instalação do wikijs 2.5.297 no ubuntu server 22.04

### DISCUSSION THREAD

**Discussion by Gustavim22** [requarks/wiki#6201] [Help / Questions]
Estou com o seguinte problema, ao realizar a configuração e startar dentro do ( diretório wiki executando o node server) ele inicia normalmente e consigo acesso via http, porém ao fechar o ssh o serviço fica indisponível, alguém já passou por isso e conseguiria me ajudar ?

Usando o ubuntu server 22.04
postgresql 9.6

**Comment by cdn0x12** [comment]
You can refer to the **run as service** section of this doc page:
https://docs.requarks.io/install/linux

**Reply by cdn0x12** [reply]
Additional info is needed to determine the problem, please provide:
1. the full content of your `wiki.service` file
2. the `ls -al` result under your wiki's root directory
3. the result of `whereis node`

**Comment by Gustavim22** [comment]


**Reply by cdn0x12** [reply]
It seems like the directory your wiki installed is not `/var/wiki`, please try to:
1. Change the `WorkingDirectory=/var/wiki` to `WorkingDirectory=<path of your wiki>`
2. Change the `User=nobody` to `User=wiki` (Assuming that the owner of your wiki installation directory is wiki)

Then `systemctl daemon-reload` and `systemctl start wiki`, if error occurs, please attach the logs(using `journalctl -u wiki`)

**Comment by Gustavim22** [comment]


**Reply by cdn0x12** [reply]
I still suspect it's something wrong with your `User` configuration.

Your previous `ls -al` screenshot corresponds to the **parent** directory of your wiki.js instance rather than the exact(root)  directory. So I can't tell which directory your wiki.js is installed in. I assume you installed wiki in the `wiki` directory owned by user `wiki` (rather than the `wikijs` directory owned by user `wikijs`), thus asked you to change the `User=nobody` to `User=wiki`.

If you are sure that your `User` and `WorkingDirectory` configuration in your `wiki.service` file is correct, you can then try to execute `systemctl daemon-reexec` and `systemctl start wiki`.

You could also try to change `User` directly to `root`, although this may cause other permission/security related problems.

**Comment by Gustavim22** [comment]
![image](https://user-images.githubusercontent.com/126171070/221180377-3fc25387-9a5e-4939-8f6f-1167b94cf901.png)
Consegue me auxiliar sobre o erro que está dando agora ? se seria algum erro na configuração ou se seria questão só de permissão mesmo ?

**Reply by cdn0x12** [reply]
Your latest screenshot shows that you mistyped the word `Environment` and `WorkingDirectory`. Please correct it before doing any further operation.

You only showed me the unmodified `wiki.service` configuration at the beginning. I can't see the complete path of your wiki installation (as you executed `ls -al` under the parent directory rather than the root directory of your wiki installation), and I don't know the contents of your changed `wiki.service` file. Therefore, I can only propose possible solutions based on my assumptions, and can't help you determine the specific reasons.

We need to clarify the following two points:
(1) what is the absolute path of your wiki instance
(2) which user owns the directory we just mentioned in (1)
